ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_idm_api.py
         template: *ci-master-latest
         timeout: 3600
-        topology: *master_1repl_1client
+        topology: *ipaserver
```

Comment:
The « ipaserver » topology is used for webui tests and installs a server before launching ipa-run-tests.
You should rather use master_1repl topo even if the test only needs 1 server.